src/components/source_editor/LatticeConfigurationDialog.js

```diff
@@ -108,6 +109,10 @@ class LatticeConfigurationDialog extends React.Component {
         const newMaterial = new Material(newMaterialConfig);
         // assert basis is stored in 'crystal' units
         newMaterial.toCrystal();
+
+        // update to non-periodic if asked to do so
+        this.state.isNonPeriodic ? newMaterial.toCartesian() : newMaterial.toCrystal();
```

Comment:
What's the logic here??

Reply:
I don't think we actually needed this line, it was from a previous update.